components/addForm.js

```diff
@@ -0,0 +1,82 @@
+import styled from 'styled-components';
+
+export default function AddCharacterForm() {
+  return (
+    <StyledContainer>
```

Comment:
This wording is generally ok, but you could rename the components to show their purpose, for example you could call this component `FormWrapper`, `FormContainer` or `FormCard`.